### PR TITLE
Create androidProjects.yml

### PR DESCRIPTION
**Pull request by anshulkhattar**

### CODE REVIEW

File: androidProjects.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: androidProjects
```
Comment (shiftkey) [inline androidProjects.yml:1]: Could this be named something without reverting to camelcasing?

Something like `Beginner-Level-Android-Studio-Apps`?

File: androidProjects.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+desc: this repository consists of some basic projects of android studio
```
Comment (shiftkey) [inline androidProjects.yml:2]: Suggested change: `A collection of basic Android projects that are tailored to new contributors`

File: androidProjects.yml (hunk 1 above)
Comment (anshulkhattar) [reply androidProjects.yml:1]: I'm unable to change the file, the edit option is blocked.

Comment (shiftkey) [reply androidProjects.yml:1]: @anshulkhattar the fork has been deleted

**Comment by shiftkey** [comment]
@anshulkhattar as we're unable to make changes as part of the code review, I'm going to close out this PR. Please open a fresh PR and keep the fork around so we can get the PR into a state where it's good to merge.

**Pull request closed without merging**